src/mongo/db/storage/kv/dictionary/kv_sorted_data_impl.cpp

```diff
@@ -89,7 +89,7 @@ namespace mongo {
                                        const IndexDescriptor* desc)
         : _db(db),
           _ordering(Ordering::make(desc ? desc->keyPattern() : BSONObj())),
-          _dupsAllowed(false)
+          _dupsAllowed(true)
```

Comment:
@igorsol, look how Rocks implemented this thing: https://github.com/percona/mongo-rocks/blob/v3.2-fix/src/rocks_engine.cpp#L370
Maybe it's also better to go this way?